app/controllers/api/accounts_controller.rb

```diff
@@ -6,11 +6,17 @@ class Api::AccountsController < Api::BaseController
 
   represent_with Api::AccountRepresenter
 
+  private
+
   def resources
-    @accounts ||= user.try(:accounts) || Account.active.member
+    @resources ||= super.member
```

Comment:
This `#member` scope will filter out the individual account for a user, which previously would have been returned.
I don't know if that is correct, but it is a change.